pkgs/by-name/ca/cargo-aoc/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-aoc";
+  version = "0.3.7";
+
+  src = fetchFromGitHub {
+    owner = "gobanos";
+    repo = "cargo-aoc";
+    rev = version;
+    hash = "sha256-k9Lm91+Bk6EC8+KfEXhSs4ki385prZ6Vbs6W+18aZSI=";
+  };
+
+  cargoHash = "sha256-DKP9YMbVojK7w5pkX/gok4PG6WUjhqUdvTwSir05d0s=";
+
```

```suggestion

  cargoBuildFlags = [
    "--package"
    "cargo-aoc"
  ];
  cargoTestFlags = cargoBuildFlags;

```
We only need to build the main binary's crate here